frontend/index.jsx

```diff
@@ -262,39 +251,33 @@ class App extends React.Component {
           </li>
         </ul>
       </div>
-        { this.state.tab === 'my' ?
-          <div className="container">
-            <ImageWall images={this.state.images} />
-          </div> :
           <div className="container">
-            <ImageWall images={this.state.imageFeed.map(imageData => imageData.image)} />
+            <ImageWall imageInfos={this.state.index.images} show={this.state.tab=='my'}/>
+            <ImageWall imageInfos={this.state.imageFeed} show={this.state.tab=='friends'}/>
```

Comment:
The reason for this change is so that the images don't have to be reloaded every time you switch the tabl